### PR TITLE
Markdown + whitespace fixes

### PR DESCRIPTION
**Pull request by adam-nielsen**
The current page doesn't terminate the XML formatting properly, so some of the Markdown code following the XML is formatted as if it's part of the XML code block.  This whitespace change fixes that.

### CODE REVIEW

**Comment by tnguyen1** [comment]
ok to test

File: md/web-service-tutorial.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ This example process will use a Web Service connector to call a sample Web Servi
        </tns:GetWeather>
      </env:Body>
     </env:Envelope>
-    ```
```
Comment (redboul) [inline md/web-service-tutorial.md]: Actually, the opening ``` should be aligned with the ending one. 
Ideally, we would like to indent it the previous line (aligned with `Envelope`)

File: md/web-service-tutorial.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ This example process will use a Web Service connector to call a sample Web Servi
     * In Expression type, click on **Script**
     * The script should be as follows:
 
-    ```groovy
+```groovy
```
Comment (redboul) [inline md/web-service-tutorial.md]: same here, aligned with `The`

File: md/web-service-tutorial.md [hunk 3]
```diff
@@ -90,15 +90,15 @@ This example process will use a Web Service connector to call a sample Web Servi
...
         if (node.getNodeType() == Node.ELEMENT_NODE)
         {
                 String key = node.getNodeName();
                 String value = node.getTextContent();
                 data.put(key, value);
         }
-    }
+   }
    return data;
 ```
```
Comment (redboul) [inline md/web-service-tutorial.md]: same here aligned with opening ```

File: md/web-service-tutorial.md [hunk 4]
```diff
@@ -113,10 +113,10 @@ List<List<String>> table = new ArrayList<List<String>>();
...
-row.add(entry.getKey());
-row.add(entry.getValue());
-table.add(row);
+    List<String> row = new ArrayList<String>();
+    row.add(entry.getKey());
+    row.add(entry.getValue());
+    table.add(row);
 }
 return table;
 ```
```
Comment (redboul) [inline md/web-service-tutorial.md]: same here, this whole block should be aligned with the `Use`

**Review by redboul — CHANGES_REQUESTED** [review]

**Comment by redboul** [comment]
Hi Adam,

Thanks for your feedback !
Would you mind indent the content the way I suggested it please ?

Best regards

**Comment by educhastenier** [comment]
retest this please

**Comment by educhastenier** [comment]
Formatting ok like that.
Thanks for your contribution.

**Comment by adam-nielsen** [comment]
Thanks all.  @redboul Sorry I didn't get a chance to update the formatting, I got moved into a different project.

**Comment by educhastenier** [comment]
Your change is already online on version 7.3 and later here:
http://documentation.bonitasoft.com/7.3?page=web-service-tutorial :smile:
Thx

**Pull request merged**